mne/externals/decorator.py

```diff
@@ -161,7 +161,13 @@ def update(self, func, **kw):
 
     def make(self, src_templ, evaldict=None, addsource=False, **attrs):
         "Make a new function from a given template and update the signature"
-        src = src_templ % vars(self)  # expand name and signature
+        # Expand name and signature. Remove `*` argument if present
+        # (placed before keyword-only arguments in function signatures).
+        vars_ = vars(self)
+        if 'kwonlyargs' in vars_:
+            vars_['signature'] = vars_['signature'].replace('*, ', '')
+        src = src_templ % vars_
```

Comment:
In general we try just to vendor this rather than make our own updates, so we should check to see if an updated version of https://github.com/micheles/decorator/blob/master/src/decorator.py will do what we want

Reply:
So this is what happens when you don't check a file's entire path, and only look at its base name. 🤦‍♂️

Reply:
Which functionality from this package do we use that's not already in Python's standard library?

Reply:
IIRC mostly the `verbose` decorator needs it to preserve traceback information properly. It's possible it's outdated for Python 3.6+, not sure

Reply:
I've checked with the latest [`decorator`](https://github.com/micheles/decorator) release and the problem persists.